empty-plugin/src/test/resources/projects/testInstall/reference/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Would actually rather not have the reference file—just forces everyone to do every change to the archetype twice. Suffices to verify that it builds.